src/main/java/ac/grim/grimac/checks/impl/badpackets/BadPacketsH.java

```diff
@@ -0,0 +1,36 @@
+package ac.grim.grimac.checks.impl.badpackets;
+
+import ac.grim.grimac.checks.CheckData;
+import ac.grim.grimac.checks.type.PacketCheck;
+import ac.grim.grimac.player.GrimPlayer;
+import com.github.retrooper.packetevents.event.PacketReceiveEvent;
+import com.github.retrooper.packetevents.protocol.packettype.PacketType;
+import com.github.retrooper.packetevents.wrapper.play.client.WrappedPacketInUseEntity;
```

Comment:
WrappedPacketInUseEntity is the class name in PacketEvents 1.8
```suggestion
import com.github.retrooper.packetevents.wrapper.play.client.WrapperPlayClientInteractEntity;
```